include/tvm/node/container.h

```diff
@@ -82,6 +82,21 @@ class MapNode : public Object {
   /*! \brief the data content */
   ContainerType data;
 
+  /*!
+   * \brief Check if all keys in the MapNode are instance of the given Object type
+   * \tparam T The given Object type
+   * \return The result of the check
+   */
+  template <class T>
+  bool KeysAreInstance() const {
```

Comment:
This is not so commonly used, let us make it an internal helper function instead, we can also use https://en.cppreference.com/w/cpp/algorithm/all_any_none_of

Reply:
@junrushao1994 please followup

Reply:
yeah I just replaced it with std::all_of. Thanks!